bundles/org.openhab.binding.opensprinkler/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -85,6 +86,14 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+
+	<channel-type id="waterlevel">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Water level</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions